### PR TITLE
add erc20 + nft deploy tutorials

### PR DESCRIPTION
**Pull request by critesjosh**
Add tutorials that show how to easily deploy erc20 tokens and NFTs to Celo.

### CODE REVIEW

File: docs/developer-resources/walkthroughs/no-code-erc20.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+You can easily deploy token contracts that use the ERC20, ERC721, or ERC1155 token standard to Celo without having to write any code.
+
+# Deploying ERC20 Tokens on Celo (no code)
+
+In this tutorial, we will go over how to deploy an ERC20 token contract. The process is very similar for deploying other tokens as well.
+
+1. Install [Metamask](https://metamask.io/).
+2. [Add the Celo network](/getting-started/wallets/using-metamask-with-celo/manual-setup#adding-a-celo-network-to-metamask) to Metamask. I suggest adding the Alfajores testnet to Metamask as well, so you can test contract deployments before deploying to mainnet.
```
Comment (YazzyYaz) [inline docs/developer-resources/walkthroughs/no-code-erc20.md:12]: I'd change "I suggest" to "We suggest" given you are using I and We interchangeably in the doc.

File: docs/developer-resources/walkthroughs/no-code-erc20.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+3. Add a small amount of CELO to your Metamask account. In this example, we will deploy to the Alfajores testnet, so we need Alfajores CELO, which you can get from the faucet [here](https://celo.org/developers/faucet).
+4. Go to the [Open Zeppelin Contracts Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard).
+5. Select `ERC20` as the type of contract that you would like to deploy. ![erc20 empty settings.png](https://github.com/critesjosh/images/blob/main/token_deploy_tutorials/erc20%20empty%20settings.png?raw=true)
+6. Name your token. I am calling our token “ProsperityToken” in this example.
```
Comment (YazzyYaz) [inline docs/developer-resources/walkthroughs/no-code-erc20.md:16]: I to We

File: docs/developer-resources/walkthroughs/no-code-erc20.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+7. Select the features for your token. I am making ProsperityToken mintable, burnable and enabling snapshots, so the token may be used for governance. I am also making the contract Ownable, so the deployer of the contract can mint new tokens and distribute them as desired. Ideally, the owner account will be a multi-signature contract, so no single person has control over this token contract. ![erc20 filled settings.png](https://github.com/critesjosh/images/blob/main/token_deploy_tutorials/erc20%20filled%20settings.png?raw=true)
```
Comment (YazzyYaz) [inline docs/developer-resources/walkthroughs/no-code-erc20.md:17]: Same comment on I to We

**Review by ericnakagawa — APPROVED** [review]
👍

**Pull request merged**